src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -766,6 +771,29 @@ public async Task ImportFromCsv()
                                     if (ContentDispositionHeaderValue.TryParse(section.ContentDisposition, out ContentDispositionHeaderValue contentDisposition) == false)
                                         continue;
 
+                                    if (MultipartRequestHelper.HasFormDataContentDisposition(contentDisposition))
+                                    {
+                                        var key = HeaderUtilities.RemoveQuotes(contentDisposition.Name);
+                                        if (key != "CsvConfig")
```

Comment:
Please extract `"CsvConfig"` to constants

Reply:
`csvImportOptions` (to match smuggler one)

Reply:
done